google-pub/c/README.md

```diff
@@ -1,55 +1,48 @@
-# Message Queue Telemetry Transport (MQTT) client for Google Cloud IoT Core
-This sample app demonstrates connecting to Google Cloud IoT Core and publishing
-a single message.
+# Google Cloud MQTT Sample
+
+## Introduction
+This sample app demonstrates connecting to Google Cloud IoT Core and publishing a single message. This Sample is modified for ease of creating a new Intel(R) System Studio project.
+
+## Hardware requirements
+No special H/W requirements, as long as your device could be connected to the internet.
+
+## Software requirements
+This version of the sample has been tested on Ubuntu Linux.
+
+## Setup
+Refer to [this document](https://cloud.google.com/iot/docs/quickstart) on how to create deviceid, registryid, and projectid parameters as they are used in calculating the client ID used for connecting to the MQTT bridge.
 
 # Building
-To build the sample, you must first download, build, and install the following
-projects:
+To build the sample, you must first download, build, and install the following projects, you can either install from individual links or follow the below commands.
```

Comment:
Are these build steps are relevant to ISS? I believe all this should work in ISS right?
If so then lets have a section with 'Building outside ISS'  with these steps. 

Is there anything the user needs to do when building in ISS?

Reply:
I see, I will keep all the building steps in a separate section as you mentioned. I don't think the user needs to do anything when building in ISS except the credentials it asks when running the sample.